File: app/src/main/java/priv/rtbishop/coopstat/ui/SettingsFragment.java
package priv.rtbishop.coopstat.ui;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import priv.rtbishop.coopstat.R;

public class SettingsFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preference, rootKey);
    }
}

File: app/src/main/java/priv/rtbishop/coopstat/ui/SettingsFragment.kt
package priv.rtbishop.coopstat.ui

import android.os.Bundle

import androidx.preference.PreferenceFragmentCompat
import priv.rtbishop.coopstat.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.preference, rootKey)
    }
}